src/app/tests/AppTestContext.h

```diff
@@ -26,14 +26,41 @@ namespace Test {
  */
 class AppContext : public LoopbackMessagingContext
 {
-    typedef LoopbackMessagingContext Super;
-
 public:
-    /// Initialize the underlying layers.
-    CHIP_ERROR Init() override;
+    // Performs shared setup for all tests in the test suite
+    virtual CHIP_ERROR SetUpTestSuite();
+    // Performs shared teardown for all tests in the test suite
+    virtual void TearDownTestSuite();
+    // Performs setup for each individual test in the test suite
+    virtual CHIP_ERROR SetUp();
```

Comment:
General Q, not a criticism of this PR.

Is this part of the swap over to the pw unit test from nlUnitTest? If so, does the pw unit test framework support error returns on setup functions?